tests/net/socket/socketpair/testcase.yaml

```diff
@@ -18,3 +18,10 @@ tests:
     extra_configs:
       - CONFIG_PICOLIBC=y
     platform_exclude: vmu_rt1170 mimxrt1160_evk_cm7 # See #61246
+  net.socket.socketpair.high_mem:
+    extra_configs:
+      # Low buffer sizes (e.g., 8192) will verify the crash fix, but tests will still
+      # fail due to insufficient memory. So, use high buffer sizes.
+      - CONFIG_NET_SOCKETPAIR_BUFFER_SIZE=4096
+      - CONFIG_HEAP_MEM_POOL_SIZE=32768
+    platform_exclude: vmu_rt1170 mimxrt1160_evk_cm7 # See #61246
```

Comment:
This really looks like a recipe for future headache. The test will fail if some new platforms are added with limited memory. Probably better to test this with just some specific platform only, or figure out a way to test this without this kind of kludge.

Reply:
Yeah, I have used `native_posix_64` for my tests, so, I can limit it to that, but just thought running on all platforms might get some coverage. Also, the test itself isn't too useful, as it doesn't run with low memory (I tried using `console` harness to make the test fail but still twister pass, but no option to write a complex `regex`). So, I can even remove it?

Reply:
Perhaps it makes sense to remove it then.

Reply:
or we could leave the test that uses large buffers, but add `min_ram: 64` ?

Reply:
That sounds like a good plan :+1: